xu1541/misc/Makefile

```diff
@@ -60,7 +60,7 @@ LDFLAGS_EXTRA+=-s
 # add libusb
 CFLAGS  = $(CFLAGS_EXTRA) -I$(LIBUSB_DIR)/include -I../include/ -Wall
 LDFLAGS = $(LDFLAGS_EXTRA) -L$(LIBUSB_DIR)/lib
-LIBS    = -L../lib -lxu1541$(LIB_WIN) -lusb
+LIBS    = -L../lib -lxu1541$(LIB_WIN)
```

Comment:
`$LDFLAGS_EXTRA` already includes the correct `-lusb-legacy` so I'm assuming it also includes `-lusb` on other platforms and therefore this explicit `-lusb` is superfluous.